MAUI/Maps/legend.md

```diff
@@ -832,4 +832,270 @@ public class Model
+                DataSource="{Binding Data}"
+                PrimaryValuePath="State"
+                ShapeDataField="name" ShowShapeTooltip="True"
+                ShapeStroke="DarkGray" ShowDataLabels="True"
+                ShapeColorValuePath="Population">
+             <map:MapShapeLayer.DataLabelSettings>
+                 <map:MapDataLabelSettings>
+                     <map:MapDataLabelSettings.DataLabelStyle>
+                         <map:MapLabelStyle FontSize="12" TextColor="Crimson" FontAutoScalingEnabled="True"/>
+                     </map:MapDataLabelSettings.DataLabelStyle>
+                 </map:MapDataLabelSettings>
+             </map:MapShapeLayer.DataLabelSettings>
+
+             <map:MapShapeLayer.ColorMappings>
+                    <map:RangeColorMapping Color="#809fff"
+                                            From="0" To="100"
+                                            Text="0-100"/>
+                    <map:RangeColorMapping Color="#3366ff" 
+                                            From="100" 
+                                            To="500" 
+                                            Text="100-500"/>
+                    <map:RangeColorMapping Color="#0039e6" 
+                                            From="500" 
+                                            To="1000" 
+                                            Text="500-1000"/>
+                    <map:RangeColorMapping Color="#002db3" 
+                                            From="1000" 
+                                            To="5000" 
+                                            Text="1000-5000"/>
+                    <map:RangeColorMapping Color="#001a66"
+                                            From="5000"
+                                            To="50000"
+                                            Text="5000-50000"/>
+             </map:MapShapeLayer.ColorMappings>
+
+             <map:MapShapeLayer.Legend>
+                 <map:MapLegend ItemTemplate="{StaticResource legendTemplate}">
+                     <map:MapLegend.TextStyle>
+                         <map:MapLabelStyle FontSize="12" TextColor="Crimson" FontAutoScalingEnabled="True" />
+                     </map:MapLegend.TextStyle>
+                 </map:MapLegend>
+             </map:MapShapeLayer.Legend>
+             <map:MapShapeLayer.BubbleTooltipSettings>
+                 <map:MapTooltipSettings>
+                     <map:MapTooltipSettings.TextStyle>
+                         <map:MapLabelStyle FontSize="12" TextColor="Crimson" FontAutoScalingEnabled="True"/>
+                     </map:MapTooltipSettings.TextStyle>
+                 </map:MapTooltipSettings>
+             </map:MapShapeLayer.BubbleTooltipSettings>
+         </map:MapShapeLayer>
+     </map:SfMaps.Layer>
+ </map:SfMaps>
+
+{% endhighlight %}
+{% endtabs %}
+
+N> The TextColor property of the legend updates as per the luminosity.
```

Comment:
add the proper note like this

The BindingContext of the ItemTemplate is the corresponding underlying legend item provided in the MapLegned class.

Reply:
Updated the changes